src/element/typeChecks.ts

```diff
@@ -24,7 +25,8 @@ export const isGenericElement = (
     (element.type === "selection" ||
       element.type === "rectangle" ||
       element.type === "diamond" ||
-      element.type === "ellipse")
+      element.type === "ellipse" ||
+      element.type === "embeddable")
```

Comment:
not a generic element. But I also see we're not using this check anywhere, so perhaps we should remove altogether.